app/src/editor/canvas/index.jsx

```diff
@@ -572,31 +573,37 @@ const CanvasEditWrap = () => {
     )
 }
 
-const CanvasErrorPage = () => (
-    <ErrorPageContent>
-        <Button
-            kind="special"
-            tag={Link}
-            to=""
-            onClick={(event) => {
-                event.preventDefault()
-                window.location.reload()
-            }}
-        >
-            <Translate value="editor.error.refresh" />
-        </Button>
-        <Button
-            kind="special"
-            tag={Link}
-            to={routes.canvases()}
-            className="d-none d-md-inline-flex"
-        >
-            <Translate value="editor.general.backToCanvases" />
-        </Button>
-    </ErrorPageContent>
-)
-
-const CanvasContainer = withRouter(withErrorBoundary(CanvasErrorPage)((props) => (
+const CanvasErrorBoundary = ({ error }) => {
+    if (error instanceof ResourceNotFoundError) {
```

Comment:
If the resource is missing we "bubble" the error up to the error boundary. Otherwise render the regular canvas error.